crates/pop-common/src/account_id.rs

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: GPL-3.0
```

Comment:
This function + tests where in the pop-contracts crate now is used in chain functionality has to be moved in pop-common